From 697b347118f88f65839dbf93dfc583b06407e791 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Fri, 19 Jun 2015 16:41:39 +0100
Subject: [PATCH 075/204] BCM270X_DT: Document the i2s-mmap overlay

---
 arch/arm/boot/dts/overlays/README | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/overlays/README b/arch/arm/boot/dts/overlays/README
index 3e08f98..7fa6d33 100644
--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -260,6 +260,12 @@ Params: ds1307                   Select the DS1307 device
         pcf8563                  Select the PCF8563 device
 
 
+Name:   i2s-mmap
+Info:   Enables mmap support in the bcm2708-i2s driver
+Load:   dtoverlay=i2s-mmap
+Params: <None>
+
+
 Name:   iqaudio-dac
 Info:   Configures the IQaudio DAC audio card
 Load:   dtoverlay=iqaudio-dac
-- 
2.6.2

